test/dashboard-e2e/pages/CreateTestPage.ts

```diff
@@ -65,4 +77,8 @@ export class CreateTestPage {
   private async clickCreateTestButton(): Promise<void> {
     await this.page.click('button[data-test="add-a-new-test-create-button"]');
   }
+
+  private static getAbsoluteFixtureFilePath(fixtureFileName: string) {
+    return path.resolve(`fixtures/files/${fixtureFileName}`);
```

Comment:
Optionally: This would be relative from working directory while executing tests, selecting based on the current path could be more stable:
```suggestion
    return path.resolve(__dirname, `../fixtures/files/${fixtureFileName}`);
```